src/backends/mod.rs

```diff
@@ -103,6 +106,8 @@ pub fn default_output_device() -> impl AudioOutputDevice {
     return default_output_device_from(&alsa::AlsaDriver);
     #[cfg(os_coreaudio)]
     return default_output_device_from(&coreaudio::CoreAudioDriver);
-    #[cfg(os_wasapi)]
-    return default_output_device_from(&wasapi::WasapiDriver);
+    // #[cfg(os_wasapi)]
+    // return default_output_device_from(&wasapi::WasapiDriver);
+    #[cfg(os_asio)]
+    return default_output_device_from(&asio::AsioDriver::new().unwrap());
```

Comment:
I don't think we want to have ASIO in the default drivers list, most drivers are exclusive meaning this can cause all audio to cut out of their system. ASIO is specialized, so it make sense to not provide it as a default, but a specific option for users that want it.

This is probably for test purposes, so this servers more as a signpost like above :p